backend/amd64/emit.mlp

```diff
@@ -1186,8 +1218,40 @@ let emit_instr fallthrough i =
       I.cvtsi2sd  (arg i 0)  (res i 0)
   | Lop(Iintoffloat) ->
       I.cvttsd2si (arg i 0) (res i 0)
-  | Lop(Iintofvalue | Ivalueofint) ->
+  | Lop(Iintofvalue | Ivalueofint | Ivectorcast _) ->
```

Comment:
I would change `_` to the currently
only value, just to be sure we pause
and think when 256-bit values are
added.